config/locales/en.yml

```diff
@@ -62,10 +62,25 @@ en:
     reorder:
       instructions_markdown: Use the up/down buttons on the right to reorder steps, or click and hold on a step to reorder using drag and drop.
       automatically_updated_markdown: Step numbers will be automatically updated on save.
+    internal_change_notes:
+      label: Internal note
+      guidance: "Add a summary of your changes and why. For example: what’s been added, removed or replaced. Provide any context that helps understand what you’ve done. For example: a link to zendesk and trello ticket if there’s one."
```

Comment:
No biggie as there is already precedent but for long lines like this you might want to use the `>` folded scalar property from yaml so these long lines can span multiple lines: https://yaml.org/spec/1.2/spec.html#id2760844

Reply:
Updated to use multi-line markup so it's easier to read and edit.

Reply:
It doesn't appear to have been updated... maybe it was updated in the wrong commit?

Reply:
Yes, looks like you updated it in d9e7a571476336772a4ef9e25ba29887edc5a48d, ideally you would do it here rather than change it halfway through the PR.